tests/testthat/test-summarise_draws.R

```diff
@@ -68,3 +68,32 @@ test_that("summarise_draws warns if all variable names are reserved", {
   variables(x) <- ".log_weight"
   expect_warning(summarize_draws(x), "no variables with unreserved names")
 })
+
+test_that("multicore summarise_draws is identical to single-core summarise_draws 
+          including if some chunks contain no variables", {
+            set.seed(1)
+            nc <- 4
+            n <- 2
```

Comment:
I think some tests with `n > 2` would help make sure that the chunking works properly, right?